servicebuilder-activemq/src/main/java/no/obos/util/servicebuilder/mq/ActiveMqListener.java

```diff
@@ -29,6 +29,7 @@
 public class ActiveMqListener implements MessageQueueListener {
 
     private static final long REQUEUE_TIMEOUT = 1000;
+    static final int MAX_LOGGED_PER_MESSAGE = 1024;
```

Comment:
Pirk; så det er lettere å hvite hva maksen gjelder for.

```suggestion
    static final int MAX_LENGTH_PER_MESSAGE = 1024;
```